lightkurve/lightcurve.py

```diff
@@ -1181,17 +1181,39 @@ def query_solar_system_objects(self, cadence_mask='outliers', radius=None,
             DataFrame object which lists the Solar System objects in frames
             that were identified to contain SSOs.  Returns `None` if no objects
             were found.
+
+        Examples
+        --------
+        Find if there are SSOs affecting the lightcurve for the given time frame:
+
+            >>> df_sso = lc.query_solar_system_objects(cadence_mask=(lc.time.value >= 2014.1) & (lc.time.value <= 2014.9))  # doctest: +SKIP
+
+        Find if there are SSOs affecting the lightcurve for all times, but it will be much slower:
+
+            >>> df_sso = lc.query_solar_system_objects(cadence_mask='all')  # doctest: +SKIP
+
         """
         for attr in ['ra', 'dec']:
             if not hasattr(self, '{}'.format(attr)):
                 raise ValueError('Input does not have a `{}` attribute.'.format(attr))
 
+        sequence_type = None
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", category=DeprecationWarning)
+            # use the deprecated collections.Sequence to be compatible with python 2.7
+            sequence_type = collections.Sequence
```

Comment:
As of Lightkurve v2.x we no longer support Python 2, so it's not necessary to use a deprecated feature here, i.e. we can use a Python 3 solution here.  Is there a better/non-deprecated way to do this in Python 3?

Reply:
done, with `collections.abc.Sequence`